sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchAsyncClient.java

```diff
@@ -164,6 +166,41 @@ public String getEndpoint() {
         return this.endpoint;
     }
 
+    /**
+     * Creates a {@link SearchIndexBatchingAsyncClient} used to index documents for the Search index associated with
+     * this {@link SearchAsyncClient}.
+     * <p>
+     * This will use the default configuration values for {@link SearchIndexBatchingAsyncClient}, see {@link
+     * SearchIndexBatchingClientBuilder} for more information.
+     *
+     * @return A {@link SearchIndexBatchingAsyncClient} used to index documents for the Search index associated with
+     * this {@link SearchAsyncClient}.
+     */
+    public SearchIndexBatchingAsyncClient getIndexDocumentBatchingAsyncClient() {
+        return getIndexDocumentBatchingAsyncClient(null, null, null, null);
+    }
+
+    /**
+     * Creates a {@link SearchIndexBatchingAsyncClient} used to index documents for the Search index associated with
+     * this {@link SearchAsyncClient}.
+     *
+     * @param autoFlush Flag determining whether the batching client will automatically flush its document batch. If
+     * {@code null} is passed this will be set to {@code true}.
```

Comment:
https://blog.joda.org/2012/11/javadoc-coding-standards.html#:~:text=Do%20not%20use%20%40code%20for,and%20adds%20no%20real%20value.

Reply:
Updated